### DISCUSSION TITLE
Qual áreas vocês trabalhariam fora de Front-end?

### DISCUSSION THREAD

**Discussion by silvaMatheus** [frontendbr/forum#993] [Discussão]
Pessoal um topico meio off, mas quais áreas  de Tecnologia vcs iriam  e pq.  Tirando front end.

**Comment by laurindo** [comment]
Eu iria para área de Big Data.
Porque está em alta, porque as tecnologias envolvidas também são muito interessantes, por exemplo:
Tecnologias de Stream e linguagens funcionais como Scala, Akka, Spark, Kafka
É uma area com tecnologia que está em alta e que pagam muito bem.

**Comment by yanmagale** [comment]
Acredito que Iot e Machine Learning 😄

**Comment by woliveiras** [comment]
Mobile (Android :heart:), com certeza!

Talvez me envolvesse com IoT também por ser algo que eu curto bastante. :grin:

**Comment by kelciocajueiro** [comment]
Eu trabalho com back-end, mas queria atuar profissionalmente nas 3 frentes: back + front + mobile.
De resto, eu trabalharia com tudo o que propusessem de tecnologia nova. Gosto bastante de desafios.

**Comment by felipefialho** [comment]
Issue legal, hehe

Trabalharia facilmente com áreas ligadas a economia. 

E meu sonho mesmo era ser jogador de futebol.

**Comment by kvnol** [comment]
Dentro de tecnologia iria para IOT e Machine Learning, como o @yanmagale.

Já fora da tecnologia, iria para música (indo aos poucos) ou educação física.

**Comment by matheusml** [comment]
Blockchain e Machine Learning

**Comment by TiagoFuelber** [comment]
Back-end, Blockchain. Mas meu sonho mesmo era ser skatista profissional, ou surfista, ou ter uma banda famosa.

**Comment by leoamrl** [comment]
Sempre foi meu sonho trabalhar com música.
Atualmente estudo produção de música eletronica 100% como hobby.

**Comment by yanmagale** [comment]
@LFeh antes de virar dev eu tentei a carreira de jogador de futebol, frustrado por não ter bons empresários kkkk

**Comment by leandroruel** [comment]
eu tentei ser desenhista, mas pensei "vô passa fome nessa p%$*&" ai desisti...

**Comment by Cleysonlb** [comment]
Seria psicólogo, já estudo psicologia comportamental como hobby hahaha

**Comment by silvaMatheus** [comment]
Pessoal que falou que iria pra Machine Learning  qual seria o motivo ?

**Comment by igorthurow** [comment]
Eu me identifico com publicidade e propaganda, comunicação digital, design... Mas de sonho mesmo seria algo como ser músico, escritor, algo assim hahaha

**Comment by linconkusunoki** [comment]
seria legal poder ser um gamer profissional ou conseguir viver da arte vendida na praia kkkkkkk

**Comment by simaodeveloper** [comment]
Eu atualmente tenho foco em Front-End, mas tenho estudos voltados a UI e Back-End, porém tenho o objetivo de ter um sítio para construir:

- uma Micro-cervejaria;
- um Meliponário (cultivo de abelhas sem ferrão);

E fora isso juntar bastante dinheiro para ficar só "treidando" as ações do sítio hauhaiahauahs

**Comment by mahenrique94** [comment]
Trabalharia com back-end.

**Comment by israelmarmar** [comment]
Eu gostaria de trabalhar com inteligência artificial ou blockchain.

**Comment by klarkc** [comment]
Machine Learning e deep learning com certeza, acho muito louco fazer coisas como essas https://github.com/NVIDIA/FastPhotoStyle

**Comment by wesmelo** [comment]
Antes de ser Dev, eu tava atrás de ser fisioterapeuta

**Comment by doug2k1** [comment]
Desenvolvimento de jogos, que era meu hobby (quando eu tinha tempo) e foi o que me motivou a fazer faculdade.

**Comment by lhamadopagode** [comment]
Provavelmente Machine Learning. Fora da area de tech +1 pra música.

**Comment by bernardodiasc** [comment]
Agronomia, marcenaria ou artesanato...

**Comment by kivervinicius** [comment]
To contigo @bernardodiasc  kkk

**Comment by leandroruel** [comment]
@bernardodiasc marcenaria mano... dá dinheiro viu... artesanato tem que manjar muito

**Comment by AmandaFonseca** [comment]
Eu gostaria de trabalhar com back-end.

Fora da área adoraria ser uma atriz profissional(Sonho Master) ou ser uma empresária (Sonho médio) porém, se a situação apertasse muito eu ia ter que voltar para a minha antiga profissão eu era técnica de farmácia.

**Comment by zecafeitosa** [comment]
Não é tecnologia, mas iria ser fotógrafo. :D

**Comment by ninetails** [comment]
Ainda em TI: ir pra arquitetura de software ou dar aula :D

Fora de TI: Não faço idéia do que seria minha vida. Talvez eventos, talvez música eletrônica. O que a minha vida tivesse me levado.

**Comment by leandroruel** [comment]
@ninetails se nada desse certo, teria a última e viável opção: carpir um lote !

**Comment by bernardodiasc** [comment]
@leandroruel mano, carpi lote é uma das minhas alegrias offline (especialmente se o lote for meu) https://www.youtube.com/watch?v=MgI_RY1Ifog 😄

**Comment by vitocaio** [comment]
Eu iria para agile, scrum master ou agile coach.

**Comment by ninetails** [comment]
@leandroruel @bernardodiasc uaehuaehuae se eu tivesse propriedades pra carpir! Carpiria com sorrizão tb rs

**Comment by MatheusWTF** [comment]
Machine Learn ou Big Data (mais voltado pra Visualização de Dados), é o que eu estudo nas horas vagas e tenho mais prazer fora do Front-End nosso de cada dia.
